docs/generated/APPLICATION_CONFIG.md

```diff
@@ -302,6 +302,13 @@ When true, artifacts can be created, updated and deleted. Artifacts include stud
 
 *Default value:* `false`
 
+### tanagra.feature.maxChildThreads
+**optional** String
+
+The maximum number of child threads a single request can spawn. When unset, the application will default to using multi-threading. When set to 0 or 1, the application will only ever run things serially.
+
+*Environment variable:* `TANAGRA_FEATURE_MAX_CHILD_THREADS`
+
 
 
```

Comment:
The doc is not very clear: set to 0 -> no multi-hreading set to 1- also no multi threading. What is the difference? What about 2?

Reply:
Currently no difference between 0 and 1, because for export they'd be the same thing. But in the future, we could have cases where it would improve performance to run a single thread in the background.

2 = at most 2 child threads will be spawned.